jersey-servers/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
     compile "com.fasterxml.jackson.module:jackson-module-afterburner"
     compile "com.jcraft:jzlib"
     compile "com.palantir.safe-logging:safe-logging"
+    compile "com.palantir.tokens:auth-tokens"
```

Comment:
don't like this so much... not sure if there's a better place for param converter to live..?